test/visual/specs/button.gspec

```diff
@@ -0,0 +1,200 @@
+# http://galenframework.com/docs/reference-galen-spec-language-guide/#Objectdefinition
+
+@objects
```

Comment:
@NathanRodger Might want to reduce repeated tests by following an approach like this:

```
@objects
    button_*                                    .button

= Height =
    @on *
        button_*:
            css height is "56px"
```